libs/libc/tls/Make.defs

```diff
@@ -27,6 +27,8 @@ endif
 
 ifneq ($(CONFIG_TLS_ALIGNED),y)
 CSRCS += tls_getinfo.c
+else ifeq ($(CONFIG_BUILD_KERNEL),y)
```

Comment:
should we always compile tls_getinfo.c and filter out tls_get_info by macro in source code? Since tls_get_info is required in protected mode too.

Reply:
Ok, we can change more simple Make.defs, and tls_getinfo.c is always compiled.